robot/Cumulus/tests/browser/gift_entry/edit_template.robot

```diff
@@ -0,0 +1,140 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/PaymentPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Enable Gift Entry
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+&{method}         Default Value=Check
+&{check}          Default Value=abc11233
+&{date_default}   Default Value=Today
+&{custom}         Required=checked
+${amount}         50
+${msg}            Automation test
+
+*** Keywords ***
+Setup Test Data
+    &{contact} =     API Create Contact                 FirstName=${faker.first_name()}    LastName=${faker.last_name()}
+    Set suite variable    &{contact}
+    &{account} =     API Create Organization Account    Name=${faker.company()}
+    Set suite variable    &{account}
+    ${org_ns} =  Get Org Namespace Prefix
+    Set suite variable    ${org_ns}
+    ${ns} =  Get NPSP Namespace Prefix
+    Set suite variable    ${ns}
+    ${ui_date} =                     Get Current Date              result_format=%b %-d, %Y
+    Set suite variable    ${ui_date}
+    ${date} =     Get Current Date    result_format=%Y-%m-%d
+    Set suite variable    ${date}
+
+*** Test Cases ***
+
+Edit GE Template And Verify Changes
+    [Documentation]                  This test case makes edits to Default gift entry template and verifies that edits are saved 
+    ...                              and available when a Single gift or batch gift are created. 
+ 
+    [tags]                           feature:GE          W-039559   
```

Comment:
Please add the tag "Unstable"  as it is a new testcase and we can monitor before turning the tag to stable

Reply:
yes I had it added initially and will add it back again, I am just testing something